src/pas/plugins/ldap/plugin.py

```diff
@@ -399,17 +399,18 @@ def enumerateUsers(
             return default
         if not exact_match:
             for key in kw:
-                value = kw[key]
-                if not value.endswith("*"):
-                    kw[key] = value + "*"
+                if not kw[key].endswith("*"):
+                    kw[key] = kw[key] + "*"
+                if not kw[key].startswith("*"):
+                    kw[key] = "*" + kw[key]
+
         try:
             matches = users.search(
                 criteria=kw, attrlist=("login",), exact_match=exact_match
             )
-            print(kw, matches)
+            logger.debug(kw, matches)
         # raised if exact_match and result not unique.
```

Comment:
This comment is wrong since a `ValueError` is also raised when there is no match.